content/api-streamer/producer.textile

```diff
@@ -0,0 +1,239 @@
+@objc func publishData(){
+ channel.publish("example", data: "message data")
+}
+```
+
+h3(#manage). Manage
+
+After you've successfully set up your app and know how to deploy your data streams into them, you can set up products to manage those data streams. Products allow you to distribute different subsets of data streams within your apps to different consumers.
+
+In your API Streamer producer dashboard's products tab, you can create new Products. While doing so, you can select the app into which you have deployed the data streams you intend to share using that Product. You can then specify a channel namespace, which is essentially a name given to the logical subset of the data in your app that you wish you share. For example, if you are publishing transport data to one of your apps called 'Public Transport Data', you could be publishing the data related to buses into the namespace called 'bus' and the data related to trains into another namespace called 'train'. You can then create two separate products, one for each of these namespaces to be distributed to the same or different sets of consumers. 
+
+The concept of channels explained earlier brings further categorization to each of these namespaces. For instance, the bus data for London and Manchester could go in two channels 'bus:london' and 'bus:manchester'. A product created using the namespace 'bus' will give it's consumers access to all the channels that are part of that namespace, in this case 'london' and 'manchester'.
+
+<a href="/images/diagrams/api-streamer-products.png" target="_blank">
+  <img src="/images/diagrams/api-streamer-products.png" style="width: 100%" alt="Managing data streams using products">
+</a>
+
+h3(#distribute). Distribute
+
+After you've set up products to manage the data within your apps, it's time to distribute them to the consumers of data. You can do this by setting up subscriptions in your API Streamer producer dashboard.
+
+Creating new subscriptions is an easy way to give access to one or more of your products to the intended consumers of your data. At the time of setting up subscriptions, you can also specify how your consumers will be charged for the data they consume. You have the options for bearing all the charges yourself, letting Ably bill the consumers directly for what they consume or even set up a custom option where you incur their charges up to a certain limit and Ably will charge them directly for any messages consumed over that limit.
+
+<a href="/images/diagrams/api-streamer-subscriptions.png" target="_blank">
+  <img src="/images/diagrams/api-streamer-subscriptions.png" style="width: 100%" alt="Distributing your products to consumers">
+</a>
+
+After you've set up new subscriptions, your intended consumers will automatically receive an invitation via email to use the data streams they've been given access to, with the prcing agreements you've specified in the subscription.
```

Comment:
`After you've set up new subscriptions` to `After you've set up your subscription`

Reply:
`prcing` to `pricing`.